packages/Chem/src/package.ts

```diff
@@ -88,15 +88,15 @@ export function substructureFilter() {
 //input: string scaffoldMolString
 export function canvasMol(
   x: number, y: number, w: number, h: number, canvas: HTMLCanvasElement,
-  molString: string, scaffoldMolString: string | null = null) {
+  molString: string, scaffoldMolString: string | null = null): void {
   drawMoleculeToCanvas(x, y, w, h, canvas, molString, scaffoldMolString == '' ? null : scaffoldMolString);
 }
 
 
 //name: getCLogP
 //input: string smiles {semType: Molecule}
 //output: double cLogP
-export function getCLogP(smiles: string) {
+export function getCLogP(smiles: string): any {
```

Comment:
The return type should be number here, based on the comments (I also happen to know what logP is) :)